libs/meta/include/meta/type_traits.hpp

```diff
@@ -115,27 +108,12 @@ using IfIsUnsignedInteger = EnableIf<IsUnsignedInteger<T>, R>;
 template <typename T, typename R = void>
 using IfIsSignedInteger = EnableIf<IsSignedInteger<T>, R>;
 
-// template <typename T, std::size_t I, std::size_t F, typename R = void>
-// using IfIsFixedPoint = EnableIf<IsFixedPoint<T, I, F>, R>;
-template <typename T, typename R = void>
-using IfIsFixedPoint = EnableIf<IsFixedPoint<T>, R>;
-
-template <typename T, typename R = void>
-using IfIsNotFixedPoint = EnableIf<IsNotFixedPoint<T>, R>;
-
 template <typename T, typename R = void>
 using IfIsNullPtr = EnableIf<IsNullPtr<T>, R>;
 
 template <typename T, typename R = void>
 using IfIsPod = EnableIf<std::is_pod<T>::value, R>;
```

Comment:
TBD: `std::is_pod` is deprecated in C++20. There's a fallback `IsPOD` in meta/type_traits.hpp.